### PR TITLE
wip: demo repo init error

### PR DESCRIPTION
**Pull request by olizilla**
If i run master locally, then upgrade ipfs from 0.40 to 0.48 i hit a repo init error

```
Failed to initialise js-ipfs NotInitializedRepoError: Repo in path ipfs is not initialized!
    at _callee$ (http://localhost:3000/static/js/6.chunk.js:64421:19)
    at tryCatch (http://localhost:3000/static/js/8.chunk.js:5166:17)
    at Generator.invoke [as _invoke] (http://localhost:3000/static/js/8.chunk.js:5386:22)
    at Generator.prototype.<computed> [as next] (http://localhost:3000/static/js/8.chunk.js:5219:21)
    at asyncGeneratorStep (http://localhost:3000/static/js/0.chunk.js:678:24)
    at _next (http://localhost:3000/static/js/0.chunk.js:700:9)
```

License: MIT
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

**Comment by olizilla** [comment]
Of note, am posting this as an early WIP on upgrading IPFS and all the other deps, so Alex can take a look at the issue I hit.

**Comment by achingbrain** [comment]
Could you blow away your `node_modules` folder and `package-lock.json` and try again, first ensuring that you've ended up with `ipfs-repo-migrations@1.0.1` installed?

**Comment by achingbrain** [comment]
..or use `ipfs@0.48.2`

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "i18next-http-backend": "^1.0.15",
     "i18next-icu": "^1.3.1",
     "internal-nav-helper": "^1.0.2",
-    "ipfs": "^0.40.0",
+    "ipfs": "^0.48.1",
```
Comment (achingbrain) [inline package.json:26]:
```suggestion
    "ipfs": "^0.48.2",
```

**Pull request closed without merging**